addons/core/translations/en-us.yaml

```diff
@@ -32,6 +32,7 @@ titles:
   no-results: No Results
   choose-a-provider: Choose a provider
   provider: Provider
+  sync-interval-seconds: Sync Interval Seconds
```

Comment:
Can we change this title to `Sync Interval`?

And can we also add a description that says:
```
Number of seconds between the time boundary syncs hosts in this set. Uses system default if unchecked.
```

Reply:
added!